### ISSUE TITLE
UWP Support?

### ISSUE THREAD

**Issue by Wrathrowe** [mmaitre314/IpCamera#2]
Hello,

I was just wondering if you have any plans on supporting UWP?

Thank you

**Comment by mmaitre314** [comment]
No, but I accept PRs :-) An alternative might be to copy the code directly into your VS solution.

**Comment by Wrathrowe** [comment]
Thanks Matthieu, I'll give it a try!

**Comment by hereswilson** [comment]
Did anyone ever get this working for UWP?

**Comment by michaelprosario** [comment]
Hi. Is there a particular reason that the IpCamera library is coded in C++?  Is the scope of this library possible in C#?  I'm learning a lot from the study of this code.  We do appreciate your time!

**Comment by mmaitre314** [comment]
The main interest of C++ is that it gives access to extra interfaces like IBufferByteAccess that are not projected to C#. In this project that was probably not that required (C# can get around that missing interface by doing a few more memory copy). That was more required to create MediaCaptureReader and get access to the audio/video samples.
